pkg/defers/defers.go

```diff
@@ -0,0 +1,19 @@
+// Package defers allows to have "global defer", by pushing functions into this
+// packages data holder, which can then be ran from main
+package defers
+
+var defers []func()
+
+// Run should be deferred from the most outer function, to allow "global defer"
+func Run() {
+	// last in. first out
+	for ii := range defers {
+		i := len(defers) - 1 - ii
+		defers[i]()
+	}
```

Comment:
nit: IMO being explicit with a `for i := len(defers)-1; i >= 0; i--` is clearer here.